tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -710,6 +710,14 @@ module.exports = {
       await client.page.locationPicker().selectFolder(target)
 
       return this
+    },
+    clickOnFileName: async function(fileName) {
```

Comment:
does `clickRow` do the same thing?

Reply:
No it doesn't. `clickRow` opens the sidebar.